library/network/src/lib/network/susefirewall.rb

```diff
@@ -554,10 +554,10 @@ def SuSEFirewallIsInstalled
       return true if @needed_packages_installed
 
       if Mode.normal
-        @needed_packages_installed = PackageSystem.CheckAndInstallPackages([@FIREWALL_PACKAGE])
+        @needed_packages_installed = Package.CheckAndInstallPackages([@FIREWALL_PACKAGE])
         log.info "CheckAndInstallPackages -> #{@needed_packages_installed}"
       else
-        @needed_packages_installed = PackageSystem.Installed(@FIREWALL_PACKAGE)
+        @needed_packages_installed = Package.Installed(@FIREWALL_PACKAGE)
```

Comment:
good example that we should finally clean susefirewall, as it is dead technology and we use only firewalld.

Reply:
You are right, but I do not plan to do it in this PR.